### ISSUE TITLE
Update to 3.9.5

### ISSUE THREAD

**Issue by mattcg** [alephdata/aleph-helm-charts#3]
3.9.5 solves a breaking logout issue. I started working on a PR to update but am not sure about the version numbers of dependent libraries. I cannot trace the actual api version, for example.

If someone can provide some brief guidelines, I'd be happy to make the PR and document the steps in the readme so that others in the community would be able to contribute these update PRs in the future.

**Comment by sunu** [comment]
Hi @mattcg, I have updated the charts for Aleph 3.9.5. Generally for the version numbers, we'll use the latest Aleph release number for api, ui and worker (https://github.com/alephdata/aleph-helm-charts/blob/master/charts/aleph/values.yaml#L7-L9) and the latest ingest-file and convert-document hash from their respective repositories in their charts.

For this release, we changed how Oauth is configured (See https://docs.alephdata.org/developers/changelog#3-9-5). So I had to make those changes in the config map that defines Aleph's settings.
